dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSIndexTest.java

```diff
@@ -67,7 +67,7 @@ public void hpccTLKFilterTest() throws Exception
         // FieldDef[] fieldDefs = null;
         // Integer key
         FieldDef[] fieldDefs = new FieldDef[2];
-        fieldDefs[0] = new FieldDef("key", FieldType.INTEGER, "INTEGER4", 4, true, true, HpccSrcType.LITTLE_ENDIAN, new FieldDef[0]);
+        fieldDefs[0] = new FieldDef("key", FieldType.INTEGER, "INTEGER4", 4, true, false, HpccSrcType.LITTLE_ENDIAN, new FieldDef[0]);
```

Comment:
tlk fields are always signed?

Reply:
No, they can be signed or unsigned. I changed the dataset so that it could be used to test this bug.